protocols/api/src/test/java/org/apache/james/protocols/api/OIDCSASLParserTest.java

```diff
@@ -98,26 +98,10 @@ void parseShouldReturnResponseWhenPartsDoNotSort() {
         SoftAssertions.assertSoftly(softly -> {
             softly.assertThat(parseResult.isPresent()).isTrue();
             softly.assertThat(parseResult.get().getToken()).isEqualTo(TOKEN_WITHOUT_PREFIX);
-            softly.assertThat(parseResult.get().getUser()).isEqualTo(USER);
+            softly.assertThat(parseResult.get().getAssociatedUser()).isEqualTo(USER);
         });
     }
 
-    @Test
-    void parseShouldReturnEmptyWhenInputHasSeveralAuthPart() {
```

Comment:
Why is this test removed?

Reply:
When I use `StringTokenizer` for parsing parts, I see it is unnecessary,